ads/google/doubleclick.md

```diff
@@ -1,131 +0,0 @@
-<!---
-Copyright 2015 The AMP HTML Authors. All Rights Reserved.
```

Comment:
Let's remove this file completely but rather indicate that doubleclick support fast fetch (or better yet have it forward automatically) as users may have this bookmarked.

Reply:
Added the file back with a short statement and link to the main docs.